infra/bicep/core/host/containerappstub.bicep

```diff
@@ -44,76 +45,70 @@ resource containerApp 'Microsoft.App/containerApps@2024-03-01' = {
   tags: tags
   identity: {
     type: 'UserAssigned'
-    userAssignedIdentities: {
-      '${userIdentity.id}': {}
-    }
+    userAssignedIdentities: { '${userIdentity.id}': {} }
   }
   
   properties: {
-    environmentId: containerAppEnvironmentResource.id
+    managedEnvironmentId: containerAppEnvironmentResource.id
     configuration: {
       ingress: {
-        targetPort: targetPort
         external: true
+        targetPort: targetPort
+        transport: 'auto'
       }
-      secrets: [for secret in items(secrets): {
-        name: secret.key
-        identity: userIdentity.id
-        keyVaultUrl: secret.value
-      }]
       registries: [
         {
           identity: userIdentity.id
           server: '${registryName}.azurecr.io'
         }
       ]
+      secrets: [for secret in items(secrets): {
+        name: secret.key
+        identity: userIdentity.id
+        keyVaultUrl: secret.value
+      }]
     }
     template: {
-      scale: {
-        minReplicas: 1
-        maxReplicas: 10
-      }
       containers: [
         {
-          name: 'app'
+          name: workloadProfileName
           image: fetchLatestImage.outputs.?containers[?0].?image ?? 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
+          env: env
           resources: {
-              cpu: json('0.5')
-              memory: '1.0Gi'
+            cpu: json('1.0')
```

Comment:
why do we need 1 CPU for the app?

Reply:
Switching this back now that NSG has fixed the issue.